test/unittests/mongoBackend/mongoContextProvidersUpdate_test.cpp

```diff
@@ -523,20 +523,27 @@ TEST(mongoContextProvidersUpdateRequest, noPatternAttrOneSingle)
   EXPECT_EQ(SccOk, ms);
 
   EXPECT_EQ(SccNone, res.errorCode.code);
```

Comment:
Fixed (in the scope of this PR) in b8da84f. For the others, I have created this issue: https://github.com/telefonicaid/fiware-orion/issues/832

I have check that after b8da84f the test of failing test hasn't change (that's good news: I was afraid that chaning the Scc* logic could impact in serviceRoutines in some point and make more harness tests to fail).